opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -55,26 +60,92 @@
 
 _EXPORTER_OTLP = "otlp"
 _EXPORTER_OTLP_PROTO_GRPC = "otlp_proto_grpc"
+_EXPORTER_OTLP_PROTO_HTTP = "otlp_proto_http"
+
+_EXPORTER_BY_OTLP_PROTOCOL = {
+    "grpc": _EXPORTER_OTLP_PROTO_GRPC,
+    "http/protobuf": _EXPORTER_OTLP_PROTO_HTTP,
+}
+
+_EXPORTER_ENV_BY_SIGNAL_TYPE = {
+    "traces": OTEL_TRACES_EXPORTER,
+    "metrics": OTEL_METRICS_EXPORTER,
+    "logs": OTEL_LOGS_EXPORTER,
+}
+
+_PROTOCOL_ENV_BY_SIGNAL_TYPE = {
+    "traces": OTEL_EXPORTER_OTLP_TRACES_PROTOCOL,
+    "metrics": OTEL_EXPORTER_OTLP_METRICS_PROTOCOL,
+    "logs": OTEL_EXPORTER_OTLP_LOGS_PROTOCOL,
+}
 
 _RANDOM_ID_GENERATOR = "random"
 _DEFAULT_ID_GENERATOR = _RANDOM_ID_GENERATOR
 
+_logger = logging.getLogger(__name__)
+
 
 def _get_id_generator() -> str:
     return environ.get(OTEL_PYTHON_ID_GENERATOR, _DEFAULT_ID_GENERATOR)
 
 
-def _get_exporter_names(names: str) -> Sequence[str]:
-    exporters = set()
+def _get_exporter_entry_point(
+    exporter_name: str, signal_type: Literal["traces", "metrics", "logs"]
+):
+    if exporter_name not in (
+        _EXPORTER_OTLP,
+        _EXPORTER_OTLP_PROTO_GRPC,
+        _EXPORTER_OTLP_PROTO_HTTP,
+    ):
+        return exporter_name
+
+    # Checking env vars for OTLP protocol (grpc/http).
+    protocol_env_var = _PROTOCOL_ENV_BY_SIGNAL_TYPE.get(signal_type)
+    if not protocol_env_var:
+        return exporter_name
```

Comment:
When does this happen? The `signal_type` is always going to be one of {`traces`, `metrics`, `logs`}.

Reply:
This is not expected to happen, though if it does and I will ignore this case, some unrelated exception will be raised.
So this is to avoid this case by possible future changes, but not completely sure if necessary.

Reply:
I mean it's just us who are using these private methods right? We are not going to pass some signal type that is not supported or random from `_get_exporter_names` invocation.

Reply:
Skipping this check then 👍